netci.groovy

```diff
@@ -32,6 +32,7 @@ def addPullRequestJob(String project, String branch, String os, String configura
 
   addBuildStepsAndSetMachineAffinity(newJob, os, configuration);
   Utilities.standardJobSetup(newJob, project, true, "*/${branch}");
+  Utilities.setJobTimeout(newJob, 180)
```

Comment:
Semicolons? I suppose it's just for style here, inconsistent in the rest of the file anyway.

Reply:
Thanks, fixed.